src/main/java/Tars.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+
+public class Tars {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        // Opening speech.
+        System.out.println("    " + "------------------------------------------------------------");
```

Comment:
Could you store the "----------...------------" part of the output as it's own string variable and use it wherever you need it.